File: D11_APP/function/bluetooth/src/main/java/com/luxshare/bluetooth/listener/IWriteListener.java
package com.luxshare.bluetooth.listener;

/**
 * The bluetooth write listener.
 *
 * @author ChenCe
 * @version version
 */
public interface IWriteListener {

    void onWriteSuccess(); // 写成功

    void onWriteFailure(); // 写失败
}

File: D11_APP/function/ble/src/main/java/com/luxshare/ble/interfaces/IWriteListener.kt
package com.luxshare.ble.interfaces

/**
 * The bluetooth write listener.
 *
 * @author ChenCe
 * @version version
 */
interface IWriteListener {
    fun onWriteSuccess() // 写成功
    fun onWriteFailure() // 写失败
}
